http/dfly_hammer2.sh

```diff
@@ -0,0 +1,187 @@
+#!/bin/sh
+
+# Install a DragonFly system from livecd.
+#
+# Mostly taken from:
+# https://gist.github.com/liweitianux/547652a3dd3a853afed71ba50410ffb5
+#
+
+01_format_disk() {
+    local _disk=$1
+
+    dd if=/dev/zero of=/dev/$_disk bs=32k count=16
+    gpt create -f $_disk
+    gpt add -t efi     -s  262144 $_disk  # ESP (128 MB)
+    gpt add -t ufs     -s 4194303 $_disk  # UFS /boot (2 GB)
```

Comment:
For a VM, 1GB for the `/boot` should be enough. The `/boot/kernel` is currently about 600-700MB (with debug info and unstriped); the striped old kernel `/boot/kernel.old` is usually ~100MB.

I used 2GB there because I want to keep the old kernel **unstriped**, for development purpose.